services/velero/5.2.3/helmrelease/helmrelease.yaml

```
@@ -13,7 +13,7 @@ spec:
         kind: HelmRepository
         name: vmware-tanzu.github.io
         namespace: kommander-flux
-      version: 5.2.2
+      version: 5.2.3
```

```suggestion
      version: 5.2.2
```

Not bumping helm chart version